Nona-Creative/nona-playground
# chore: convert react native project to typescript

## Scope

- PT-38 - convert iso-ts-react-native to typescript

## Work Done

- convert project to typescript
- update eslint rules for typescript
- add jest watch typeahead for better test filtering
- update outdated dependencies
- update bundling
- change init ios command to use npm package pod-install.
- fix start reset command to reset metro bundler

## Steps to Test

- pull the branch
- install the dependencies and install your pods
- run the app on android and ios and make sure you get the following screenshots

## Screenshot
<img width="427" alt="ScreenCapture at Sun Nov 15 21:58:25 SAST 2020" src="https://user-images.githubusercontent.com/42996940/99195331-b5dd8e00-278d-11eb-8713-4d12c46a6efa.png">
